lib/db-utils.js

```diff
@@ -337,13 +337,45 @@ async function getScraperRunsByLocation(locationRefId) {
     return scraperRuns;
 }
 
+async function getScraperRunsByLocationsSortedByTimestamp(locations) {
+    const queries = locations.map((loc) =>
+        fq.Map(
+            fq.Paginate(
+                fq.Match(
+                    fq.Index("scraperRunsByLocationRefSortByTimestamp"),
+                    fq.Ref(fq.Collection("locations"), loc.ref.id)
+                )
+            ),
+            fq.Lambda(["ts", "ref"], fq.Get(fq.Var("ref")))
+        )
+    );
+
+    const scraperRuns = await faunaQuery(queries);
+    return scraperRuns;
+}
+
+async function getScraperRunsByLocationSortedByTimestamp(location) {
+    const query = fq.Map(
+        fq.Paginate(
+            fq.Match(
+                fq.Index("scraperRunsByLocationRefSortByTimestamp"),
+                fq.Ref(fq.Collection("locations"), location.ref.id)
+            )
+        ),
+        fq.Lambda(["ts", "ref"], fq.Get(fq.Var("ref")))
+    );
+
+    const scraperRuns = await faunaQuery(query);
+    return scraperRuns;
+}
+
 async function getScraperRunsByLocations(locations) {
     const queries = locations.map((loc) =>
         fq.Map(
             fq.Paginate(
                 fq.Match(
                     fq.Index("scraperRunsByLocation"),
-                    fq.Ref(fq.Collection("locations"), loc.refId)
+                    fq.Ref(fq.Collection("locations"), loc.ref.id)
```

Comment:
Changed this so that I can pass in documents that come right from the Locations collection. 
And I'm eventually going to remove the test that previously relied on this.